### PR TITLE
Docs: Update readme with more details

### PR DESCRIPTION
**Pull request by hellofanny**
## What's the purpose of this pull request?

- Added examples summary and references.
- Adds how to use and section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,10 +11,89 @@ This project serves as a compilation of examples that highlight the fundamental
...
-### Naming Conventions
+## 👩‍🏫 How to use this repository
+
+The repository contains folders/files with code [examples](#-examples) for the features mentioned above.
+
+**Follow Along with Documentation**:
+As you run the examples, refer to the documentation to understand the concepts and implementation details of each part of the code. The documentation will guide you through the expected output and any configurations needed.
+
+**Experiment and Modify**: Feel free to experiment with the code examples by making modifications. This will help you gain a deeper understanding of how each feature works.
+
```
Comment (Mariana-Caetano) [inline README.md:29]:
```suggestion

### Cloning the repository

```

File: README.md [hunk 2]
```diff
@@ -11,10 +11,89 @@ This project serves as a compilation of examples that highlight the fundamental
...
+Clone the repository in your local machine and start exploring:
+
+1. Clone the repository to your local machine:
```
Comment (Mariana-Caetano) [inline README.md:32]:
```suggestion
1. Open your terminal and clone the repository to your local machine by running the following command:
```

File: README.md [hunk 3]
```diff
@@ -11,10 +11,89 @@ This project serves as a compilation of examples that highlight the fundamental
...
+
+```
+git clone https://github.com/vtex-sites/playground.store.git
+```
+
+2. Navigate to the repository directory:
```
Comment (Mariana-Caetano) [inline README.md:38]: Before this step, should we recommend the person create a new branch?

**Review by Mariana-Caetano — COMMENTED** [review]
Awesome, Fanny! ✨
I left some suggestions.

File: README.md (hunk 3 above)
Comment (hellofanny) [reply README.md:40]: I don't think its necessary, because they probably won't send anything to this repo 🤔

**Review by Mariana-Caetano — APPROVED** [review]

**Review by emersonlaurentino — APPROVED** [review]

**Review by eduardoformiga — APPROVED** [review]

**Pull request merged**